Directory.Build.props

```diff
@@ -27,8 +27,8 @@
 
     <IsPublishable>false</IsPublishable>
 
-    <!-- Set this manually whenever build ARM64-->
-    <Arm64Build>false</Arm64Build>
+    <!-- Set the target platform manually to be x86 (default), x64, arm64 -->
+    <TargetPlatform Condition=" '$(TargetPlatform)' == '' ">x86</TargetPlatform>
```

Comment:
IMO it should be inverted; x86 is a thing of a past, the default should be x64.
```suggestion
    <!-- Set the target platform manually to be x64 (default), x86, arm64 -->
    <TargetPlatform Condition=" '$(TargetPlatform)' == '' ">x64</TargetPlatform>
```

Reply:
Yes, I agree that we should move forward and default to `x64` Windows platform, but this comes with a catch: the resulting `.msi` installer package will refuse to install in x86 (i.e. 32-bit) Windows box. Is this restriction acceptable?

With the default `x86` platform, the .msi package will be installable on all 3 Windows platforms: x86, x64 and Arm64; it's the package of the "lowest common denominator." But I guess at some point we need to cut off this `x86` support...

Anyway, the AppVeyor build script could be changed to generate packages for both `x86` and `x64` platforms at the same build time:
```
dotnet publish -c Release /p:TargetPlatform=x86
dotnet publish -c Release /p:TargetPlatform=x64
```
and the user can pick whatever .msi package they want to install.

Reply:
IIRC, there was some flag in the isntaller that forced user to manually uninstall the old app first then install the new one. This way we could migrate the user to the new 64bit app.

Reply:
Have a look into Setup/EnableUpgrades.wxi `var.MinSupportedVersion`

Reply:
I think all these extra requirements (default x64, upgrade strategy, minimum support version) should belong to a different PR than this one, as they require far too much work/consideration as compared to my original intention for this PR.

Reply:
Sure